### ISSUE TITLE
Imperial Federation

### ISSUE THREAD

**Issue by clementou** [SighPie/HFM#96]
Give Britain Cores on Dominions through Danubian Federation-like election

Tag switch?

**Comment by AFP595** [comment]
Wouldn't perhaps _"The (British) Commonwealth"_ fit better for such a hypothetical union?

**Comment by clementou** [comment]
Yes, but the [Imperial Federation](https://en.wikipedia.org/wiki/Imperial_Federation) is the historical one.
